src/components/CardGame/index.js

```diff
@@ -0,0 +1,13 @@
+import React, {Component} from 'react';
+import './index.scss'
+
+class CardGame extends Component{
```

Comment:
Esto podría ser un componente estático, seria algo asi..

```suggestion
const CardGame = (props) => {

return (
 ...
)
```